### DISCUSSION TITLE
Trying to use SDRPLAY (RSP2Duo)

### DISCUSSION THREAD

**Discussion by uakm2201** [rsadsb/dump1090_rs#103] [Q&A]
Hello

I'm trying to use a RSP2Duo without success 5I'm able to see around 30 planes using an oldest dump1090)

No ioda about the parameter I should use ....

I've not more text than this one ...

[-] read in custom config: /home/pi/config.toml
[-] using soapysdr driver: driver=sdrplay
[INFO] devIdx: 0
[INFO] SerNo: 1708004110
[INFO] hwVer: 2
[INFO] rspDuoMode: 0
[INFO] tuner: 1
[INFO] rspDuoSampleFreq: 0.000000
[-] using config: Sdr {
    channel: 0,
    driver: "sdrplay",
    setting: None,
    gain: [
        Gain {
            key: "LNA",
            value: 0.0,
        },
        Gain {
            key: "VGA",
            value: 0.0,
        },
    ],
    antenna: None,
}
[-] Writing gain: LNA = 0
[-] Writing gain: VGA = 0
[-] frequency: Ok(1090000000.0)
[-] sample rate: Ok(2400000.0)
[INFO] Using format CS16.

**Comment by wcampbell0x2a** [comment]
Have you tried changing the gain values? For instance, the following is used for the HackRF: https://github.com/rsadsb/dump1090_rs/blob/417f90c7a123ba153a87289beb5bc06defa0a74d/dump1090_rs/config.toml#L11
